app/Livewire/Edition/EditEditionModal.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Livewire\Edition;
+
+use App\Livewire\Forms\EditionForm;
+use App\Models\Edition;
+use Illuminate\View\View;
+use LivewireUI\Modal\ModalComponent;
+
+class EditEditionModal extends ModalComponent
+{
+    public Edition $edition;
+    public EditionForm $form;
+
+    /**
+     * Initializes the component
+     * @param Edition $edition
+     * @return void
+     */
+    public function mount(Edition $edition): void
+    {
+        $this->edition = $edition;
+        $this->form->setEdition($edition);
+    }
+
+    /**
+     * Saves the updates made on the form
+     */
+    public function save()
```

Comment:
Maybe authorization? I was thinking that a policy can be implemented over here and overall just `EditionPolicy` to be created

Reply:
Yeah, good idea! I will implement it then.